core-web/libs/sdk/client/src/lib/client/content/builders/collection/collection.ts

```diff
@@ -222,12 +250,19 @@ export class CollectionBuilder<T = unknown> {
     }
 
     /**
-     * Takes a string that represents a variant ID of content created with the {@link https://www.dotcms.com/docs/latest/experiments-and-a-b-testing A/B Testing} feature
-     *
-     * `variantId` defaults to "DEFAULT" to fetch content that is not part of an A/B test
+     * Filters the content by a variant ID for A/B testing.
```

Comment:
```suggestion
     * Filters the content by a variant ID for Experiments.
```

Add a link to the experiments docs.